services/backend/src/database/seeds/README.md

```diff
@@ -20,22 +18,22 @@ const path = require("path");
 const prisma = require("../..");
 const data = require("./data");
 
-async function seedData() {
-  const setupData = []; // Add prisma api calls to this array
-  return Promise.all(setupData);
+async function seedStaticInfo() {
+  const staticInfo = []; // add the prisma function here
+  return Promise.all(staticInfo);
 }
 
 async function seed() {
   const folderName = path.dirname(__filename).split(path.sep).pop();
 
-  const dbSeed = await prisma.dbSeeds.findOne({
+  const dbSeed = await prisma.dbSeed.findUnique({
     where: { id: folderName },
   });
 
-  if (dbSeed) {
+  if (!dbSeed) {
```

Comment:
what this logic suppose to be reversed?

Reply:
Yes, it was I think it was a typo in the original readme but we should only seed if it's not found in the database